### ISSUE TITLE
lpad-align breaks npm audit

### ISSUE THREAD

**Issue by MulMic** [kevva/lpad-align#5]
On my last npm upgrade I ended up with 2 vulnerabilities, that won't fix, no matter what.

As it turns out lpad-align insists on pulling in meow@3.7.0

please update, thx

**Comment by karlhorky** [comment]
More information here: https://github.com/kevva/squeak/issues/4

**Comment by carpogoryanin** [comment]
meow fixed it in v10.1.1: https://github.com/sindresorhus/meow/commit/a0daf207695c5caa438f39168308ca71984f6b8c
Please update meow in package.json at least this way:
```"meow": "^10.0.0"```
instead of current:
```"meow": "^9.0.0"```

**Comment by karlhorky** [comment]
Seems like #9 updates to `meow@10`

**Comment by Eskaan** [comment]
Yeah, that's exactly what I've done. So my pr fixes 2/3 issues on this repo lol
